### PR TITLE
Removed unused mlo's

### PR DESCRIPTION
**Pull request by Zeeroqt**
all of these mlo's that have been removed are not used or that are fps eater

### CODE REVIEW

File: server.cfg [hunk 1]
```diff
@@ -109,9 +109,9 @@ ensure venice-beach-house
 ensure mrp_house
 #ensure usarrp-bikergang
 #ensure reapers
-ensure vinewood_house
-ensure int_weed
-ensure rockford_records
+#ensure vinewood_house
+#ensure int_weed
```
Comment (minipunch) [inline server.cfg:113]: Bruh 1) this is a fire interior and unless you can show me more proof that it is 'unoptimized' like the server console saying a specific file is too large or smth then let's keep this for now

and 2) if we did want to remove it it has a weed process location we'd need to remove also

File: server.cfg [hunk 2]
```diff
@@ -65,7 +65,7 @@ ensure gabz_lost
 start map_dealership
 ensure sandy_so_hospital
 start map_burgershot
-start map_fightclub
+#start map_fightclub
```
Comment (minipunch) [inline server.cfg:68]: there is a map blip here that u should remove if u remove this

**Review by minipunch — CHANGES_REQUESTED** [review]
see comments

File: server.cfg (hunk 2 above)
Comment (Zeeroqt) [reply server.cfg:68]: yeah pushing now

File: server.cfg (hunk 1 above)
Comment (Zeeroqt) [reply server.cfg:113]: it is in the smack middle of legion which there are 4 different mlo's in legion. which can cause it to have a drop of frames. I can remove it from the fire script and the weed process also.

File: server.cfg (hunk 2 above)
Comment (Zeeroqt) [reply server.cfg:68]: https://github.com/minipunch/usarealismrp-fxserver/pull/1839/commits/2f3322457fec4e674fc68a7b12d2feabab59846c done

**Comment by Zeeroqt** [comment]
i dont i can just make them in the same pr because when i do it on github desktop it includes my server key, steam api key, and other stuff. unless there is a way i can just add it to this one

**Comment by minipunch** [comment]
> i dont i can just make them in the same pr because when i do it on github desktop it includes my server key, steam api key, and other stuff. unless there is a way i can just add it to this one

**Comment by Zeeroqt** [comment]
> ****
> [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)

i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.

**Comment by Zeeroqt** [comment]
> > [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)
> 
> i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.

i mean i cant have it in the same pr on github bc it leaks the stuff i stated above

**Comment by minipunch** [comment]
> > > [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)
> > 
> > 
> > i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.
> 
> i mean i cant have it in the same pr on github bc it leaks the stuff i stated above

but I'm saying u should be able to without leaking anything lol

**Comment by Zeeroqt** [comment]
> > > > [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)
> > > 
> > > 
> > > i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.
> > 
> > 
> > i mean i cant have it in the same pr on github bc it leaks the stuff i stated above
> 
> but I'm saying u should be able to without leaking anything lol

it says +sv_licensekey" then my key" imma edit it on github and then put stuff back after

**Comment by minipunch** [comment]
> > > > > [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)
> > > > 
> > > > 
> > > > i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.
> > > 
> > > 
> > > i mean i cant have it in the same pr on github bc it leaks the stuff i stated above
> > 
> > 
> > but I'm saying u should be able to without leaking anything lol
> 
> it says +sv_licensekey" then my key" imma edit it on github desktop by deleting my stuff and adding it back

Sounds like you skipped the `server_internal.cfg` file part of the readme 😂

**Comment by Zeeroqt** [comment]
ok new pr done

**Comment by Zeeroqt** [comment]
> > > > > > [ Resolve conflicts ](/minipunch/usarealismrp-fxserver/pull/1835/conflicts)
> > > > > 
> > > > > 
> > > > > i cant in the same pr when i do the changes it will list my server key, my steam web api key, and more. So i had to edit the server.cfg on github through browser. Unless u can merge 2 things in the same PR through github online too.
> > > > 
> > > > 
> > > > i mean i cant have it in the same pr on github bc it leaks the stuff i stated above
> > > 
> > > 
> > > but I'm saying u should be able to without leaking anything lol
> > 
> > 
> > it says +sv_licensekey" then my key" imma edit it on github desktop by deleting my stuff and adding it back
> 
> Sounds like you skipped the `server_internal.cfg` file part of the readme 😂

closing this in 5 min

**Pull request closed without merging**